app/views/govuk_publishing_components/components/_notice.html.erb

```diff
@@ -8,29 +8,35 @@
   local_assigns[:margin_bottom] ||= 8
   local_assigns[:margin_bottom] = 8 if local_assigns[:margin_bottom] > 9
 
+  banner_title ||= "Important"
```

Comment:
Could we translationify this?

Reply:
Have moved the string to the locale file